rules/windows/registry/registry_set/registry_set_abusing_windows_telemetry_for_persistence.yml

```diff
@@ -1,4 +1,4 @@
-title: Abusing Windows Telemetry For Persistence
+title: Abusing Windows Telemetry For Persistence - Registry Set
```

Comment:
`title: Abusing Windows Telemetry For Persistence - Registry`
Like the other rules
need to fix the 2 last 
- rules/windows/registry/registry_set/registry_set_renamed_sysinternals_eula_accepted.yml:title: Usage of Renamed Sysinternals Tools - RegistrySet
- rules/windows/registry/registry_set/registry_set_uac_bypass_eventvwr.yml:title: UAC Bypass via Event Viewer - Registry Set

Reply:
Modified